src/state/accountSelectors.ts

```diff
@@ -687,3 +691,73 @@ export const getUserWalletAddress = (state: RootState) => state.account.wallet?.
 
 export const getUserSubaccountNumber = (state: RootState) =>
   state.account.subaccount?.subaccountNumber;
+
+export const getAccountUiMemory = (state: RootState) => state.accountUiMemory;
+export const getCurrentAccountMemory = createAppSelector(
+  [getSelectedNetwork, getUserWalletAddress, getAccountUiMemory],
+  (networkId, walletId, memory) => memory[walletId ?? '']?.[networkId]
+);
+
+export const createGetUnseenOrdersCount = () =>
```

Comment:
seems like we use quite a few selector prefixes
- get
- select
- create
- calculate

What is your mental model to differentiate how we should be naming these?

Reply:
I think we should do select() for all selectors and createSelect for selector creators like this one

Reply:
calculate is for pure functions, as is get

Reply:
for now I'll leave these matching the rest of the file but eventually we should standardize.

Reply:
Sounds good! Thanks for the explanation